### DISCUSSION TITLE
Audio recording available

### DISCUSSION THREAD

**Discussion by Anonymouse83** [Motion-Project/motion#1768] [General]
I believe this has been raised before, but couldn't find the thread to add to it.
I appreciate that in some places it may be illegal to capture audio, but maybe in one's home it is OK - or maybe have it disabled by default, and a warning before enabling, which puts the burden on the user.
Or maybe you could point us to where in the code the change would be required - at our risk... That would be really appreciated.
I looked at using motionplus - however, it uses a new set of libraries that I cannot install, so cannot use motionplus - otherwise, that may have been a solution!
Many thanks,

**Comment by tosiara** [comment]
Motion does not record audio
